cluster/centos/build.sh

```diff
@@ -24,84 +24,84 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-readonly ROOT=$(dirname "${BASH_SOURCE}")
-source ${ROOT}/config-build.sh
+readonly ROOT=$(dirname "${BASH_SOURCE[0]}")
+source "${ROOT}"/config-build.sh
 
 # ensure $RELEASES_DIR is an absolute file path
-mkdir -p ${RELEASES_DIR}
-RELEASES_DIR=$(cd ${RELEASES_DIR}; pwd)
+mkdir -p "${RELEASES_DIR}"
+RELEASES_DIR=$(cd "${RELEASES_DIR}"; pwd)
 
 # get absolute file path of binaries
-BINARY_DIR=$(cd ${ROOT}; pwd)/binaries
+BINARY_DIR=$(cd "${ROOT}"; pwd)/binaries
 
 function clean-up() {
-  rm -rf ${RELEASES_DIR}
-  rm -rf ${BINARY_DIR}
+  rm -rf "${RELEASES_DIR}"
+  rm -rf "${BINARY_DIR}"
 }
 
 function download-releases() {
-  rm -rf ${RELEASES_DIR}
-  mkdir -p ${RELEASES_DIR}
+  rm -rf "${RELEASES_DIR}"
+  mkdir -p "${RELEASES_DIR}"
 
   echo "Download flannel release v${FLANNEL_VERSION} ..."
-  curl -L ${FLANNEL_DOWNLOAD_URL} -o ${RELEASES_DIR}/flannel.tar.gz
+  curl -L "${FLANNEL_DOWNLOAD_URL}" -o "${RELEASES_DIR}"/flannel.tar.gz
```

Comment:
same here and anywhere else where the variable is referring to a file path -- quote the entire file path, variable included. it's all supposed to be a single string that you're passing